### PR TITLE
Fix CI not catching errors when failing unit tests.

### PR DESCRIPTION
**Pull request by larroy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-280-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-280/3/index.html

**Comment by szha** [comment]
Job PR-280-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-280/4/index.html

File: README.md [hunk 1]
```diff
@@ -27,3 +27,4 @@ See the [AutoGluon Website](http://autogluon.mxnet.io/index.html) for instructio
 ## License
 
 This library is licensed under the Apache 2.0 License.
+
```
Comment (zhanghang1989) [inline README.md]: Why adding an empty line here

Comment (Innixma) [reply README.md]: We should end all files with a new line, thats what this adds.

**Pull request merged**